vizro-core/src/vizro/models/_navigation/_accordion.py

```diff
@@ -76,7 +76,7 @@ def _get_accordion_container(self, accordion_items, accordion_buttons):
                 html.Hr(),
             ],
             className="nav_panel",
-            id=f"{self.id}_outer",
+            id="nav_panel_outer",
```

Comment:
Is this so they can be adressed all together, and not require a one-by-one treatment?

Reply:
The nav panel is always unique per page, so they don't require an additional provision of a unique ID. This also makes it easier for us and the user to affect the container, as the ID is always the same. Previously users wouldn't even be able to provide an ID to the nav_panel without accessing our private models. 

If at some point we could have several nav panels within the same page, we would need the ID in there, but currently, this is not planned.

Reply:
Following that logic, I would need to add "nav_bar_outer" to the nav_bar to my icon pr